addon/bootstrap.js

```diff
@@ -59,7 +61,59 @@ const appStartupObserver = {
   }
 }
 
+const LibraryButton = {
+  ITEM_ID: "appMenu-library-screenshots",
+
+  init(webExtension) {
+    this._initialized = true;
+    let permissionPages = [...webExtension.extension.permissions].filter(p => (/^https?:\/\//i).test(p));
```

Comment:
It would be cool if this worked for local development as well, where the server URL [is `http://localhost:10080`](https://github.com/mozilla-services/screenshots/blob/master/addon/webextension/manifest.json.template#L66). Could you change the regex?